src/js/containers/covid19/Covid19Container.jsx

```diff
@@ -121,14 +127,39 @@ const Covid19Container = () => {
                 console.log(' Error Overview : ', e.message);
             }
         };
+        const getAllAwardTypesAmount = async () => {
+            const params = {
+                filter: {
+                    def_codes: defCodes.map((code) => code.code)
+                }
+            };
+            awardAmountRequest.current = fetchAwardAmounts(params);
+            awardAmountRequest.current.promise
+                .then((res) => {
```

Comment:
to pass the eslint camelcase r ule:

```suggestion
                .then(({
                  data: {
                    obligation,
                    outlay,
                    award_count: awardCount,
                    face_value_of_loan: faceValueOfLoan
                  }
                }) => {
```